Bottom: 74a6f58f266d41a55245584586f64ee24b7bb104
Top:    ae51c0a94c97aa971d586de0d27afa49ca5d4d41
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 16:36:48 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index e877766..20bff57 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1108,24 +1108,6 @@ pad_callback (const BoxType * b, void *cl)
 static int
 hole_callback (const BoxType * b, void *cl)
 {
-#if 0
-  PinTypePtr pin = (PinTypePtr) b;
-  int plated = cl ? *(int *) cl : -1;
-
-  switch (plated)
-    {
-    case -1:
-      break;
-    case 0:
-      if (!TEST_FLAG (HOLEFLAG, pin))
-	return 1;
-      break;
-    case 1:
-      if (TEST_FLAG (HOLEFLAG, pin))
-	return 1;
-      break;
-    }
-#endif
   DrawHole ((PinTypePtr) b);
   return 1;
 }
